streams/test-utils/src/main/java/org/apache/kafka/streams/processor/MockProcessorContext.java

```diff
@@ -208,7 +210,12 @@ public MockProcessorContext(final Properties config, final TaskId taskId, final
         this.taskId = taskId;
         this.config = streamsConfig;
         this.stateDir = stateDir;
-        this.metrics = new StreamsMetricsImpl(new Metrics(), "mock-processor-context-virtual-thread");
+        final MetricConfig metricConfig = new MetricConfig();
+        metricConfig.recordLevel(Sensor.RecordingLevel.DEBUG);
```

Comment:
I think it should be fine to set the recording level to debug for all unit tests, rather than create a way to configure it.

Reply:
I agree